pet-race-devops/docker/cassandra/Dockerfile

```diff
@@ -12,26 +12,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.4
+FROM ingenieux/alpine-java8-supervisord
```

Comment:
This is based on alpine 3.2, which will not resolve dns in K8s.  We need to be off of 3.4 which fixes alpine dns issues.